src/redux/api/subscriptionApi.ts

```diff
@@ -19,7 +22,18 @@ export const subscriptionApi = baseApi.injectEndpoints({
             ]
           : ['SubscriptionHistory'],
     }),
+    getLatestSubscription: builder.query<LatestSubscriptionResponse, void>({
+      query: () => ({
+        url: '/subscriptions/latest/',
+        method: 'GET',
+      }),
+      providesTags: (result) => [
+        { type: 'LatestSubscription', id: result?.id },
+        'LatestSubscription',
+      ],
```

Comment:
From the documentation, it's always a great idea to check first whether result variable exist or not. You can refer it here:

https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#providing-cache-data